<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSystemDialog</class>
 <widget class="QDialog" name="FileSystemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="m_splitterHor">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="m_splitterVer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeView" name="m_tree">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
      </widget>
      <widget class="QTableView" name="m_table">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
      </widget>
     </widget>
     <widget class="QListView" name="m_list">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_labFileName">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labFileSize">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labFileType">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_checkDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_labFilePath">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_tree</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>m_table</receiver>
   <slot>setRootIndex(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tree</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>m_list</receiver>
   <slot>setRootIndex(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
